ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/custom-dns/private/cucushift-installer-rehearse-aws-ipi-custom-dns-private-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-custom-dns-private
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-custom-dns-private-provision
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-private-deprovision
```

Comment:
Per https://github.com/openshift/release/pull/67158, do you mind to append `- ref: junit2reportportal` here ?